OpenSim/Common/CMakeLists.txt

```diff
@@ -13,7 +13,8 @@ endif()
 OpenSimAddLibrary(
     KIT Common
     AUTHORS "Clay_Anderson-Ayman_Habib-Peter_Loan"
-    LINKLIBS ${Simbody_LIBRARIES} ${BTK_LIBRARIES}
+    # Clients of osimCommon need not link to BTK.
+    LINKLIBS PUBLIC ${Simbody_LIBRARIES} PRIVATE ${BTK_LIBRARIES}
```

Comment:
Yes; we require 3.2, which includes PUBLIC/PRIVATE. Here are the 3.1 docs: https://cmake.org/cmake/help/v3.1/command/target_link_libraries.html?highlight=target_link_libraries